src/modules/oh-indicators-list/store/slice.ts

```diff
@@ -6,23 +6,30 @@ import {
   connectionOpened,
   getAllItems,
   receiveMessage,
+  startConnection,
 } from './actions';
 
 export interface IState {
   byId: Record<string, IOpenhabItem>;
   ids: string[];
-  isConnected: boolean;
-  error: string | null;
+  sseConnection: {
+    connecting: boolean;
+    state: boolean;
```

Comment:
Consider adding is/has/do/etc. prefix to the boolean values. What does state bool mean?